lib/download.php

```diff
@@ -11,8 +11,11 @@ class DownloadUtility
 {
     public static function getDownloadFile($filename = '', $rewrite = true)
     {
+        // prevent path traversal
+        $filename = basename($filename);
+        
         if ($rewrite) {
-            $link = '/download/'.$filename;
+            $link = rex_url::base() . download/'.$filename;
```

Comment:
Hab das leider ungetestet gemerged und jetzt sind mir die Seiten um die Ohren geflogen. Fehlt ein '

Reply:
Um syntax errors im PR schon zunerkennen kann man z.b. Mit travis arbeiten
https://github.com/redaxo/redaxo/blob/962aca96c15ec9f1db7b57b8be87eb9d0fc80b73/.travis.yml#L24

Reply:
Ich kenne Travis von vielen Projekten, hab ich aber noch nie eingesetzt. Kann man das für FOR irgendwie einrichten?

Reply:
Das kann man pro repos einrichten. Für opensource projekte auch kostenlos